src/integrationTest/java/uk/gov/hmcts/reform/dataextractor/DbTest.java

```diff
@@ -50,9 +50,15 @@
                             .build(),
                     insertInto("child")
                             .columns("id", "name", "data", "created_date", "parent_id")
-                            .values(1, "A1", "{\"a1Data\": {\"name\": \"A1\", \"subject\": {\"name\": {\"lastName\": \"A1S\", \"firstName\": \"Harry\"}, \"address\": \"a1Address\" }}}::json", "2019-04-12 23:45:46", 1)
-                            .values(2, "A2", "{\"a2Data\": {\"name\": \"A2\", \"subject\": {\"name\": {\"lastName\": \"A2S\", \"firstName\": \"Harry\"}, \"address\": \"a2Address\" }}}::json", "2019-04-12 23:45:47",1)
-                            .values(3, "B1", "{\"b1Data\": {\"name\": \"B2\", \"subject\": {\"name\": {\"lastName\": \"B1S\", \"firstName\": \"Harry\"}, \"address\": \"b1Address\" }}}::json", "2019-04-12 23:45:46", 2)
+                            .values(1, "A1", "{\"a1Data\": {\"name\": \"A1\", \"subject\": "
```

Comment:
might make sense to extract to a file? this isn't very readable or maintainable with all the escaping